src/kademlia/dht_storage.cpp

```diff
@@ -453,6 +464,15 @@ namespace
 			touch_item(i->second, addr);
 		}
 
+		int get_infohashes_sample(entry& item) const override
```

Comment:
if you are talking about the pattern of passing an entry reference, yes, all over the interface